dayinfo.py

```diff
@@ -78,11 +78,12 @@ def dayinfo(event):
     args = event.args
     if len(args) == 1:
         try:
-            today = datetime.strptime(event.args[0], DATE_FORMAT)
+            today_s = datetime.strptime(event.args[0], DATE_FORMAT)
+            today = date(today_s.year, today_s.month, today_s.day)
         except ValueError:
             return 'Not a valid date'
     else:
-        today = datetime.today()
+        today = date.today()
```

Comment:
@Adman wouldn't it be better to have this just below like 78? It does not seem that the else part is really necessary.